src/components/Table/components/EditColumnsButton.tsx

```diff
@@ -76,22 +74,27 @@ export function EditColumnsButton<R extends Kinded>(props: EditColumnsButtonProp
     <OverlayTrigger {...props} menuTriggerProps={menuTriggerProps} state={state} buttonRef={buttonRef} {...tid}>
       <div
         css={{
-          ...Css.bgWhite.py5.px3.maxwPx(380).bshBasic.$,
+          ...Css.dg.gtc("1fr auto").gap2.bgWhite.p2.maxwPx(326).$,
           "&:hover": Css.bshHover.$,
         }}
       >
-        <div css={Css.gray500.xsSb.mb1.ttu.$}>{title || "Select columns to show"}</div>
-        <CheckboxGroup
-          label={title || "Select columns to show"}
-          onChange={(values) => setSelectedValues(values)}
-          values={selectedValues}
-          options={options}
-          columns={2}
-          labelStyle="hidden"
-        />
-        <div css={Css.mt1.$}>
-          <Button variant={"tertiary"} label={"Clear selections"} onClick={() => setSelectedValues([])} />
-        </div>
+        {options.map((option) => (
+          <Fragment key={option.value}>
+            <div css={Css.sm.truncate.pr1.$}>{option.label}</div>
+            <Switch
+              compact
+              selected={selectedValues.includes(option.value)}
+              onChange={(value) =>
+                setSelectedValues(
+                  value ? [...selectedValues, option.value] : selectedValues.filter((v) => v !== option.value),
+                )
+              }
+              labelStyle="hidden"
```

Comment:
`labelStyle=hidden`s are a trigger for me, that we're using/abusing components without outside of their intended usage. Would be curious if a labelstyle=left or something would work but totally get we're following a figma where everything is a snowflake.

Reply:
...doubling down on this, if the built-in/default label style flips the toggle to be on the right, that is a win imo b/c there is a reason the original UX designer picked "toggles on the right" for this component, and I'd assert the column drop-down is going "off style" without a good reason too.

Happy to start a discussion/figma thread with Andrea about my proposal, if you're not comfortable with "just doing it" -- lmk either way. :-)

Reply:
It could be a "me" problem but the issue i was running into was with our `left` labelStyle was I had to set a defined width on the container and it kept coming out either squished together with the label/switch or it looked silly with a huge gap between the the label and the toggle so i went with the happy medium.

This doesn't match the [designs either](https://www.figma.com/design/OfJJm1CXHRGkrHNcPu2grY/2025-UX-Standardization?node-id=556-6076&m=dev) but neither do our current switch components. Fonts are different and the switch is too large 🤭

Reply:
ill take another look

Reply:
Yeah this would likely be a new `wide` or `menu` labelStyle. It's really hard to style these to be both aligned and take up the space of the container without leaving extra room using just the component and maybe a div wrapper for width

Or maybe we could use the `labelLeftFieldWidth` and set that to be an `fr` instead